### PR TITLE
Docs:  Update the Elasticsearch data source docs

### PR DESCRIPTION
**Pull request by lwandz13**
Updated the Elastic search data source docs:

created a new, separate configure the data source doc that includes all 10.x UI updates
updated the query editor
cleaned up the main Elasticsearch data source page

Addresses the following issues:

https://github.com/grafana/grafana/issues/67628
https://github.com/grafana/grafana/issues/70173
https://github.com/grafana/grafana/issues/70175
https://github.com/grafana/grafana/issues/70176
https://github.com/grafana/grafana/issues/70177
https://github.com/grafana/grafana/issues/70178

### CODE REVIEW

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 1]
```diff
@@ -15,64 +15,144 @@ labels:
...
+  - geo hash grid - see [Geohash grid aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-geohashgrid-aggregation.html).
+  - date histogram - for time series queries. See [Date histogram aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html).
+  - histogram - Depicts frequency distributions. See [Histogram aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html).
+  - nested (experimental) - See [Nested aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-nested-aggregation.html).
 
-## Name a time series
+- **@timestamp** - Depicts timestamp data options. Set this when configuring the Elasticsearch data source in the **Time field name** under the [**Elasticsearch details**](/docs/grafana/latest/datasources/elasticsearch/configure-elasticsearch-data-source/#elasticsearch-details) section.
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:90]:
```suggestion
- **Time field** - Depicts date data options. The default option can be specified when configuring the Elasticsearch data source in the **Time field name** under the [**Elasticsearch details**](/docs/grafana/latest/datasources/elasticsearch/configure-elasticsearch-data-source/#elasticsearch-details) section. Otherwise **@timestamp** field will be used as a default option. 
```

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 2]
```diff
@@ -15,64 +15,144 @@ labels:
...
+  - **Timezone** - Select a timezone from the dropdown menu. The default is `Coordinated universal time`.
 
-## Control pipeline metrics visibility
+Click the **+ sign** to add multiple group by options. The data will grouped in order (first by, then by).
 
-Some metric aggregations, such as _Moving Average_ and _Derivative_, are called **Pipeline** aggregations.
-Elasticsearch pipeline metrics must be based on another metric.
-Use the eye icon next to the metric to prevent metrics from appearing in the graph.
-This is useful for metrics you only have in the query for use in a pipeline metric.
+{{< figure src="/static/img/docs/elasticsearch/group-by-then-by-10.2.png" max-width="850px" class="docs-image--no-shadow" caption="Group by options" >}}
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:101]: All of these options (starting with time field and ending with timezone) are specifically for `Date Histogram` , so if we want to add documentation for them, I think they should be under `date histogram`. 

Different types of bucket aggregations have different options. So for example `terms` accepts any field (not timestamp) and has different options such as `Order`, `Size`, ...
image

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 3]
```diff
@@ -15,64 +15,144 @@ labels:
...
+### Logs query type
 
-## Create a query
+Logs queries analyze Elasticsearch log data. You can configure the following options:
 
-Write the query using a custom JSON string, with the field mapped as a [keyword](https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html#keyword) in the Elasticsearch index mapping.
+- **Logs Options/Limit** - Limits the number of logs to analyze. The default is `500`.
 
-If the query is [multi-field](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) with both a `text` and `keyword` type, use `"field":"fieldname.keyword"` (sometimes `fieldname.raw`) to specify the keyword field in your query.
+#### Logs volume panel
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:109]: I would mention that Logs volume panel and Logs panel are available in Explore. 

In dashboard, when you run logs query, you need to select logs panel as visualisation. Moreover, in dashboard, you can't select logs volume panel.

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 4]
```diff
@@ -15,64 +15,144 @@ labels:
...
+
+- **Pretty JSON** - Toggle to pretty print JSON data to make it easier to read.
+
+- **Deduplication** - Makes repetitive log data easier to read. `Exact` matches are done on the whole line with the exception for date fields. `Numbers` matches are done on the line after stripping out numbers such as durations, IP addresses, and so on. `Signature` is the most aggressive deduplication as it strips all letters and numbers and matches on the remaining whitespace and punctuation. The default is `None`.
+
+- **Display results** - Change the order of received logs from the newest first (descending order) to oldest first (ascending order). The default is `Newest first`.
+
+- **Download** - Click to download log results in either `txt`or `json` format.
+
+Click on a specific log to get an expanded details view of each log, which provides additional information including `fields` and `links` attached to the log lines. See [Log details view](/docs/grafana/latest/explore/logs-integration/#log-details-view) for more information.
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:135]: I am wondering if it is enough to just have a link to `Logs in Explore` and not to duplicate the documentation for Logs option here as well. I am worried that it might get out of sync.

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 5]
```diff
@@ -15,64 +15,144 @@ labels:
...
+
+### Raw data query type
+
+Run a raw data query to retrieve a table of all fields that are associated with each log line.
+
+- **Raw data size** - Number of raw data log lines. You can specify a different amount. The default is `500`.
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:141]:
```suggestion
- **Raw data size** - Number of raw data documents. You can specify a different amount. The default is `500`.
```

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 6]
```diff
@@ -15,64 +15,144 @@ labels:
...
+{{% admonition type="note" %}}
+The option to run a **raw document query** is deprecated as of Grafana v10.1.
+{{% /admonition %}}
+
+## Use template variables
+
+You can also augment queries by using [template variables]({{< relref "./template-variables/" >}}).
+=======
 Queries of `terms` have a 500-result limit by default.
 To set a custom limit, set the `size` property in your query.
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:152]: This is not related to template variables.

**Review by ivanahuckova — COMMENTED** [review]
Really nice job with this! 🚀 Left some comments/questions bellow.

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+  - elasticsearch
+  - guide
+  - data source
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure Elasticsearch
+title: Configure the Elasticsearch data source
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:17]: Is it possible to add "Grafana" to the title? That'll help SEO.

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+# Configure the Elasticsearch data source
+
+Grafana ships with built-in support for Elasticsearch.
+You can make many types of queries to visualize logs or metrics stored in Elasticsearch, and annotate graphs with log events stored in Elasticsearch.
+
+For general documentation on querying data sources in Grafana, see [Query and transform data]({{< relref "../../panels-visualizations/query-transform-data" >}}).
+
+For instructions on how to add a data source to Grafana, refer to the [administration documentation]({{< relref "../../administration/data-source-management/" >}}).
+Only users with the organization administrator role can add data sources.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:29]: This is a good "Before you begin" prereq.

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+1. Select **Elasticsearch data source**.
+1. Click **Create a Elasticsearch data source** in the upper right.
+
+You will be taken to the **Settings** tab where you will set up your Elasticsearch configuration.
+
+## Configuration options
+
+The following is a list of configuration options for Elasticsearch.
+
+The first option to configure is the name of your connection:
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:48]: I'm not sure this is necessary? It's not wrong, it's just that the user can see that **Name** is the first config option so not sure if it's valuable to point that fact out. A pure nit on my part :)

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+Use TLS (Transport Layer Security) for an additional layer of security when working with Elasticsearch. For information on setting up TLS encryption with Elasticsearch see [Configure TLS](https://www.elastic.co/guide/en/elasticsearch/reference/8.8/configuring-tls.html#configuring-tls). You must add TLS settings to your Elasticsearch configuration file **prior** to setting these options in Grafana.
+{{% /admonition %}}
+
+- **Basic authentication** - The most common authentication method. Use your `data source` user name and `data source` password to connect.
+
+- **With credentials** - Toggle to enable credentials such as cookies or auth headers to be sent with cross-site requests.
+
+- **TLS client authentication** - Toggle to use client authentication. When enabled, add the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server.
+
+- **With CA cert** - Toggle to authenticate with a CA certificate. Follow the instructions of the CA (Certificate Authority) to download the certificate file.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:76]: Should this be "Toggle to authenticate with a Certificate Authority (CA). Follow the instructions of the CA to download..."?

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+
+- **Skip TLS verify** - Toggle on to bypass TLS certificate validation.
+
+- **Forward OAuth identity** - Forward the OAuth access token (and also the OIDC ID token if available) of the user querying the data source.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:80]:
```suggestion
- **Forward OAuth identity** - Forward the OAuth access token (and the OIDC ID token if available) of the user querying the data source.
```

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+
+- **Value** - The value of the header.
+
+### Elasticsearch details
+
+The following settings are specific to the Elasticsearch data source.
+
+- **Index name** - Use the index settings to specify a default for the `time field` and your Elasticsearch index's name. You can use a time pattern, such as `YYYY.MM.DD`, or a wildcard for the index name.
+
+- **Pattern** - Select the matching pattern if using one in your index name. Options include:
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:94]: Are these options visible in the UI? If so, I would remove the bulleted list. Documenting lists of values can become a maintenance issue (if, for instance, a 15 min interval is added to the UI).

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 13]
```diff
@@ -0,0 +1,153 @@
...
+  - no pattern
+  - hourly
+  - daily
+  - weekly
+  - monthly
+  - yearly
+
+- **Time field name** - Name of the time field. The default value is @timestamp. You can enter a different name.
+
+- **Max concurrent shard requests** - Sets the number of shards being queried at the same time. The default is `5``. For more information on shards see [Elasticsearch's documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/scalability.html#scalability).
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:105]:
```suggestion
- **Max concurrent shard requests** - Sets the number of shards being queried at the same time. The default is `5`. For more information on shards see [Elasticsearch's documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/scalability.html#scalability).
```

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 14]
```diff
@@ -0,0 +1,153 @@
...
+Frozen indices are [deprecated in Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/frozen-indices.html) since v7.14.
+{{% /admonition %}}
+
+### Logs
+
+In this section you can configure which fields the data source uses for log messages and log levels.
+
+- **Message field name:** - Grabs the actual log message from the default source.
+
+- **Level field name:** - Name of the field that has log level/severity information. When a level label is specified, the value of this label is used to determine the log level and update color of each log line accordingly. If the log doesn’t have specified level label, we try to find out if its content matches any of the [supported expressions](/docs/grafana/latest/explore/logs-integration/#log-level). The log level is always determined by the first match. In the case where Grafana is not able to infer a log level field, it will be visualized with an unknown log level.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:139]:
```suggestion
- **Level field name:** - Name of the field with log level/severity information. When a level label is specified, the value of this label is used to determine the log level and update the color of each log line accordingly. If the log doesn’t have a specified level label, we try to determine if its content matches any of the [supported expressions](/docs/grafana/latest/explore/logs-integration/#log-level). The first match always determines the log level. If Grafana cannot infer a log-level field, it will be visualized with an unknown log level.
```

Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:139]: Who is "we"? Are we talking about Grafana? Or Grafana Labs?

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+
+### Data links
+
+Data links create a link from a specified field that can be accessed in Explore's logs view. You can add multiple data links
+
+Each data link configuration consists off:
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:145]:
```suggestion
Each data link configuration consists of:
```

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 16]
```diff
@@ -0,0 +1,153 @@
...
+
+- **Field** - Sets the name of the field used by the data link.
+
+- **URL/query** Sets the full link URL if the link is external. If the link is internal, this input serves as a query for the target data source.<br/>In both cases, you can interpolate the value from the field with the `${__value.raw }` macro.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:149]:
```suggestion
- **URL/query** - Sets the full link URL if the link is external. If the link is internal, this input serves as a query for the target data source.<br/>In both cases, you can interpolate the value from the field with the `${__value.raw }` macro.
```

Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:149]: why use `</br>` here?

Comment (chri2547) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:149]: why use </br> here?

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 17]
```diff
@@ -0,0 +1,153 @@
...
+
+- **URL Label** (Optional) Sets a custom display label for the link. The link label defaults to the full external URL or name of the linked internal data source and is overridden by this setting.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:151]:
```suggestion
- **URL Label** (Optional) - Sets a custom display label for the link. The link label defaults to the full external URL or name of the linked internal data source and is overridden by this setting.
```

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 18]
```diff
@@ -15,64 +15,144 @@ labels:
...
+  - percentiles - see [Percentiles aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/search-aggregations-metrics-percentile-aggregation.html)
+  - unique count - see [Cardinlaity aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/search-aggregations-metrics-cardinality-aggregation.html)
+  - top metrics - see [Top metrics aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/search-aggregations-metrics-top-metrics.html)
+  - rate - see [Rate aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/8.9/search-aggregations-metrics-rate-aggregation.html)
 
 You can select multiple metrics and group by multiple terms or filters when using the Elasticsearch query editor.
 
-Use the plus and minus icons to the right to add and remove metrics or group by clauses.
-To expand the row to view and edit any available metric or group-by options, click the option text.
+Use the **plus icon** to the right to add multiple metrics to your query. Click on the **eye icon** next to "Metric" to hide metrics, and the **garbage can icon** to remove metrics.
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:79]: These steps get lost when not a numbered list.

File: docs/sources/datasources/elasticsearch/_index.md [hunk 19]
```diff
@@ -19,17 +20,15 @@ weight: 325
 
 # Elasticsearch data source
```
Comment (chri2547) [inline docs/sources/datasources/elasticsearch/_index.md:21]: Reviewed this PR and my general impression is that there is an awful lot of assumed knowledge to get this working. There's tons of terminology that's not explained - or maybe it is explained in a concept topic? They aren't necessarily terms that we own per se, but I wonder if there is a way to be up front about the level of expertise required to be successful. I would give thought to using a Before you begin section where you can articulate exactly what is required of a user in terms of knowledge, skills, and abilities.

Before you begin:
- Ensure that you are familiar with how ElasticSearch is deployed in your environment, including:
   - Refresh intervals
   - This thing, 
   - that thing
- Ensure that xxx

I'm not sure what all of those things are, but providing a list to the user so that they can quickly review it before they dive in would be helpful.

**Review by chri2547 — COMMENTED** [review]
Some suggestions and comments. Thanks!

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 20]
```diff
@@ -15,64 +15,144 @@ labels:
...
+
+There are several different types of queries you can create using the Elasticsearch query editor. The following options are available for all query types.
+
+### Add query
+
+Regardless of query type, you can create multiple queries by clicking **+ Add query**.
+
+### Inspector
+
+Click **Inspector** to get detailed statistics regarding your query. Inspector functions as a kind of debugging tool that "inspects" your query. It provides query statistics under **Stats**, request response time under **Query**, data frame details under **{} JSON**, and the shape of your data under **Data**.
```
[the remark this answers is not in the document]
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/query-editor/index.md:52]: Since they are part of the query editor they should be included.

File: docs/sources/datasources/elasticsearch/query-editor/index.md (hunk 4 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/query-editor/index.md:135]: Now that there is a dedicated data source writer I think it's ok to leave this here. I'm setting up a cadence to review data source documentation periodically.

File: docs/sources/datasources/elasticsearch/query-editor/index.md (hunk 2 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/query-editor/index.md]: I will add options for each type of group by (bucket aggregation) option - I totally missed this so thank you for pointing this out. :-)

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md (hunk 7 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:17]: I haven't added Grafana to any of the config docs...it's not part of the template I am creating but I can review that.  If I was doing a search on "configure Elasticsearch in Grafana" this page is the first one would come up, I think.

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md (hunk 10 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:76]: change made

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md (hunk 7 above)
Comment (chri2547) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:17]: All good. I got to chatting with Isabel earlier today, and we discussed the same thing wrt to viz docs but decided against it. If it shows up first, perfect. No need to consider any further! Thanks!

File: docs/sources/datasources/elasticsearch/_index.md (hunk 19 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/_index.md:21]: thanks for this feedback, I am going to try to incorporate this in the intro doc and add to the template I am creating

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md (hunk 14 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:139]: fixed

File: docs/sources/datasources/elasticsearch/query-editor/index.md (hunk 2 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/query-editor/index.md]: added group by/bucket aggregations fileds

**Comment by lwandz13** [comment]
@ivanahuckova Please review the group by/bucket aggregations after Date Histogram for accuracy

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md [hunk 21]
```diff
@@ -0,0 +1,155 @@
...
+
+- **Level field name:** - Name of the field with log level/severity information. When a level label is specified, the value of this label is used to determine the log level and update the color of each log line accordingly. If the log doesn’t have a specified level label, we try to determine if its content matches any of the [supported expressions](/docs/grafana/latest/explore/logs-integration/#log-level). The first match always determines the log level. If Grafana cannot infer a log-level field, it will be visualized with an unknown log level.
+
+- **Level field name:** - Name of the field that has log level/severity information. When a level label is specified, the value of this label is used to determine the log level and update color of each log line accordingly. If the log doesn’t have specified level label, Grafana tries to find out if its content matches any of the [supported expressions](/docs/grafana/latest/explore/logs-integration/#log-level). The log level is always determined by the first match. In the case where Grafana is not able to infer a log level field, it will be visualized with an unknown log level.
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:141]: This part is duplicated

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 22]
```diff
@@ -15,59 +15,164 @@ labels:
...
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:52]:
```suggestion
Click **Query inspector** to get detailed statistics regarding your query. Inspector functions as a kind of debugging tool that "inspects" your query. It provides query statistics under **Stats**, request response time under **Query**, data frame details under **{} JSON**, and the shape of your data under **Data**.
```

This was recently renamed to `Query inspector` to be consistent between Explore and dashboard https://github.com/grafana/grafana/issues/73603

File: docs/sources/datasources/elasticsearch/query-editor/index.md [hunk 23]
```diff
@@ -15,59 +15,164 @@ labels:
...
-| `{"find": "fields", "type": "keyword"}`                             | Returns a list of field names with the index type `keyword`.                                                                                                                  |
-| `{"find": "terms", "field": "hostname.keyword", "size": 1000}`      | Returns a list of values for a keyword using term aggregation. Query will use current dashboard time range as time range query.                                               |
-| `{"find": "terms", "field": "hostname", "query": '<Lucene query>'}` | Returns a list of values for a keyword field using term aggregation and a specified Lucene query filter. Query will use current dashboard time range as time range for query. |
+### Logs query type
+
+Logs queries analyze Elasticsearch log data. You can configure the following options:
+
+- **Logs Options/Limit** - Limits the number of logs to analyze. The default is `500`.
+
+#### Logs volume panel
```
Comment (ivanahuckova) [inline docs/sources/datasources/elasticsearch/query-editor/index.md:135]: I still think that this whole section `Logs volume panel` and `Logs panel` should not be here, cause it is not related to query editor. We don't do it for metrics or raw query results. And I think it could be confusing for everyone who uses Elasticsearch in dashboard/alerting, as `Logs volume panel` and `Logs panel` are just visualisations used in Explore.

Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/query-editor/index.md:135]: Because it's there when you open the query editor, we should document what it is.  I personally am not sure query editor docs should be under data sources, as it's basically clicking on Explore, and Explore is documented elsewhere.

File: docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md (hunk 21 above)
Comment (lwandz13) [reply docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md:141]: thanks for catching that!  removed

File: docs/sources/datasources/elasticsearch/query-editor/index.md (hunk 23 above)
Comment (ivanahuckova) [reply docs/sources/datasources/elasticsearch/query-editor/index.md:135]: But if you open Elasticsearch editor in Dashboard, it is not going to be there - see https://play.grafana.org/d/play-elastic-web-logs/elasticsearch-8-web-logs?orgId=1&editPanel=1
[image](https://play.grafana.org/d/play-elastic-web-logs/elasticsearch-8-web-logs?orgId=1&editPanel=2)

**Comment by lwandz13** [comment]
@ivanahuckova I made requested changes, can you approve so I can get this published?

**Review by Eve832 — APPROVED** [review]

**Review by ivanahuckova — APPROVED** [review]
This is great! Thank you so much for making ES docs awesome! 🚀
